ReplayMessenger.device.lib.nut

```diff
@@ -93,6 +109,41 @@ class ReplayMessenger {
         return _mm.send(messageName, data, null, _retryInterval, metadata);
     }
 
+    function onReplayBegin(loggerName, callback = null) {
+      if (callback == null && typeof loggerName == "string") { //de-register callback
+        if (loggerName in _onReplayBegin) {
+          delete _onReplayBegin[loggerName]
+        }
+      } else if (callback == null) { //assume callback is provided for "default" logger
```

Comment:
Oops - good catch. I think the comment describes what the purpose is, no? If the callback is "null" and the typeof the loggerName is *not* a string (i.e. it is a function), then the user is trying to provide a callback for the "default" logger. I'll update the code to make things clearer.